book/website/ethical-research/ethics-open-source-governance.md

```diff
@@ -0,0 +1,99 @@
+For instance, is your governance structure built to allow the whole community to participate and influence a project, that will in turn influence themselves and their work?
+
+It may feel like a giant leap to go from, say, a free-form do-ocracy to something more structured, or to give up your position of benevolent dictator, or to introduce different voting rules for the steering committee, but if you want your project to reflect your ethics, you will need to allow your ethics to impact the project, and iterate on this process. 
+It is a bit like putting debugging checkpoints in the way you run your project: run, but halt to reflect if you encounter a state that is unethical, based on your definition of ethical. 
+
+In short: `if (ethical == true) { // Do stuff } else { // FIXME }`
+
+(er-ethics-open-source-governance-action)=
+## Ethics Need Governance to become Actionable
+Open source project communities may already have an implicit set of ethics regarding their project in place. 
+However, unless these ethics are being made explicit, and inform a structured model of governance, they will remain inconsequential or, as Jo Freeman has put it in her essay [“The Tyranny of Structurelessness”](https://www.google.com/url?q=https://www.jofreeman.com/joreen/tyranny.htm&sa=D&source=docs&ust=1652716021793901&usg=AOvVaw2m6_oS7n1u7IvfHamGJDBs), impotent.
+
+In order to empower your project to act on your ethics, it will need some governance model to adhere to. 
+There already exist a number of more-or-less established governance models, and you can take a look at the resources section of this chapter for links to more information on them. 
+Keep in mind, though, that not all of them may cater well for an ethical framework, or the specific ethical framework you have in mind, in fact….
+
+(er-ethics-open-source-governance-technology)=
+## Do Not Rely on Technology to do Your Ethics for You
+
+In an age of automation, of bots, and of exceptional investment into AI, you might be inclined to think “This all sounds like a heavy workload - how do I even get started?” You might consider a couple of options - 
+
+1. Do I adopt an existing governance model? 
+2. Maybe I should get a computer system to do it for me, or at least help me out?
+
+Pulling a governance system out of the box is a great way to get started, but don’t assume that because a governance system has been in use before - perhaps even successfully - that the governance system has all the ethical kinks worked out, or even all the practical challenges. 
+For instance, the “benevolent dictator for life” (BDFL) open source governance model has practical as well as ethical issues: what if that BDFL has an unexpected health crisis and disappears? 
+How do we ensure that the dictator stays benevolent?
+What if they’re missing viewpoints that were obvious if only the governance committee was more than one person?
```

Comment:
Possible examples here

- [Linux code of conduct and Linus taking a break](https://itsfoss.com/linux-code-of-conduct/)
- [Richard Stallman using position to restore abortion joke in glibc documentation](https://www.theregister.com/2018/05/09/gnu_glic_abort_stallman/)